src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    private String password;
+
+    private String email;
```

Comment:
The 'email' field should also have an @Column annotation, and you might want to ensure it's unique by setting 'unique = true' within the @Column annotation.